src/newSeed/stage1.html

```diff
@@ -28,12 +38,19 @@
         </div>
         <template repeat.for="link of seedConfig.general.customLinks">
           <div class="otherLinks">
-            <div class="sMedia">
-              <label for="$index">Platform </label>
+            <div>
+              <div class="sMedia">
+                <label for="$index">Platform </label>
+                <span>
+                  ${link.media.length || 0}/20
```

Comment:
Font color and position are wrong here

![image](https://user-images.githubusercontent.com/1821666/124467616-543a5600-dd55-11eb-92a0-bdef145d0f4d.png)